lib/local-links-manager/export/bad_links_url_and_status_exporter.rb

```diff
@@ -14,9 +15,9 @@ def self.local_authority_bad_homepage_url_and_status_csv
         end
       end
 
-      def self.bad_links_url_and_status_csv
+      def self.bad_links_url_and_status_csv(with_ga_headings = false)
```

Comment:
The tests suggest you want this as a keyword arg (you're doing `exporter.bad_links_url_and_status_csv(with_ga_headings: true)`) but this is a normal positional arg.  The reason is works is that the arg in the test is turned into a hash, so `with_ga_headings` ends up being `{with_ga_headings: true}` which _is_ a truthy value, so we get the right output, but for the wrong reasons.

Reply:
Fixed.